lib/busapi.js

```diff
@@ -95,6 +96,29 @@ export default (services) => () => {
           let dirs = data['bustime-response'].dir;
           return dirs;
         })
+    },
+    getStopsByRoute: (rt, dir) => {
```

Comment:
Could also use this to get nested stops under the route type but the dir argument might make that tricky.